x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/job_details_step/components/advanced_section/components/model_plot/model_plot_switch.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState, useContext, useEffect } from 'react';
+import { EuiSwitch } from '@elastic/eui';
+import { JobCreatorContext } from '../../../../../job_creator_context';
+import { isPopulationJobCreator } from '../../../../../../../common/job_creator';
+import { Description } from './description';
+
+export const ModelPlotSwitch: FC = () => {
+  const { jobCreator, jobCreatorUpdate } = useContext(JobCreatorContext);
+  const [modelPlotEnabled, setModelPlotEnabled] = useState(jobCreator.modelPlot);
+
+  const disableSwitch = isPopulationJobCreator(jobCreator);
```

Comment:
Is this a deliberate change in functionality to the current population wizard, where model plot can be enabled for population jobs but a warning is displayed if enabling for a config where the cardinality of fields is greater than 100?

Reply:
When i originally checked the original population wizard this checkbox was disabled, so i copied that.
but i realise now that it was disabled because i'd not selected a field.

Reply:
I'll enable the checkbox in this PR. The cardinality check will come as part of the basic inline validation that I'll be adding next.
https://github.com/elastic/kibana/projects/30#card-23805746